dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -974,20 +1025,6 @@ sources:
                     description: user3 (Renovated) should be '0' or '1'
           - name: user4
             description: '{{ doc("shared_column_char_tp_dsgn") }}'
-            tests:
-              - not_null:
-                  name: iasworld_dweldat_char_tp_dsgn_not_null
```

Comment:
Our docs describe this field as deprecated, and it's no longer visible in iasWorld as far as I can tell, so Tia and I agreed to remove its tests.